src/backend/chat.nim

```diff
@@ -80,15 +80,15 @@ proc sendChatMessage*(
     }
   ])
 
-proc sendImages*(chatId: string, images: var seq[string]): RpcResponse[JsonNode] {.raises: [Exception].} =
+proc sendImages*(chatId: string, images: var seq[string], text: string): RpcResponse[JsonNode] {.raises: [Exception].} =
   let imagesJson = %* images.map(image => %*
       {
         "chatId": chatId,
         "contentType": 7, # TODO how do we unhardcode this
         "imagePath": image,
         # TODO is this still needed
         # "ensName": preferredUsername,
-        "text": "Please upgrade your status version to view images"
+        "text": text
```

Comment:
i18n of this text -> `qsTr("Image received")` will not work as expected because the receiver may use a different language than the sender.

Me (Polish): `sendImages` with text: "Otrzymano obrazek"
Receiver (English): will popup "Otrzymano obrazek" in notification, instead of "Image received" 

Please consider using 🖼️ emoji as text instead, as @0x-r4bbit smartly suggested.

Reply:
Ah, you're right, I haven't thought of this :/

Reply:
Ok, so it looks like translation should be provided on the receiver side.
The is known and we have a ticket for this https://github.com/status-im/status-desktop/issues/8002
Meanwhile, I'll revert changes related to passing text field

Reply:
@osmaczko reverted

Reply:
@osmaczko reverted